sdk/storage/Azure.Storage.Files.Shares/tests/FileTestBase.cs

```diff
@@ -43,7 +43,31 @@ public ShareClientOptions GetOptions()
                     MaxDelay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.1 : 10)
                 },
                 Transport = GetTransport()
-        };
+            };
+
+            if (Mode != RecordedTestMode.Live)
+            {
+                options.AddPolicy(new RecordedClientRequestIdPolicy(Recording), HttpPipelinePosition.PerCall);
+            }
+
+            return Recording.InstrumentClientOptions(options);
+        }
+
+        public ShareClientOptions GetOptions(ShareClientOptions.ServiceVersion serviceVersion)
```

Comment:
Because BlobClientOptions.LatestVersion is internal, and not accessible to the Blob Batch project.  When I make it visible Blob Batch, there is a compile error in Blob Tests as to which Latest Version to use, Blob or Blob Batch.  This is me side-stepping the issue I mentioned to you on Teams yesterday.

I chose to use the same logic in Files.